modules/javascript/pages/react.adoc

```diff
@@ -1,56 +1,48 @@
 = React Native
 :page-partials:
+:page-edition: Public Preview
 :page-toclevels: 2@
-:description: Using Couchbase Lite with Javascript applications
+:description: The React Native plugin for Couchbase Lite is now in public preview. This is an open-source, community-supported project developed by Couchbase.
 
 include::partial$_set_page_context_for_javascript.adoc[]
 
-:param-abstract: If you are building mobile or desktop applications using web technologies such as JavaScript/HTML/CSS, there are options for using Couchbase Lite as an embedded database within your app.
-:param-related: xref:ionic.adoc[] | xref:cordova.adoc[]
-include::{root-partials}_show_page_header_block.adoc[]
+== Overview
 
+Couchbase Lite Enterprise Edition for React Native is a Native Module implementation using TypeScript.
+This public preview release enables developers using React to integrate Couchbase Lite into their mobile applications, ensuring they can build offline-first, cross-platform solutions with ease.
 
-:tutorial: https://github.com/couchbaselabs/userprofile-couchbase-mobile-reactnative/blob/main/README.md
+More information on React Native Native Modules can be found https://cbl-reactnative.dev/[here].
 
-:docs: https://reactnative.dev/docs/native-modules-intro
+== Important Notes
 
-:ref_implementation: https://github.com/couchbaselabs/couchbase-lite-react-native-module/
+* Platform Support - This plugin currently only supports iOS and Android platforms. Web, Windows, and macOS are not currently supported.
 
-NOTE: Couchbase Lite for React Native is a community-driven project without official support from Couchbase.
+* Couchbase Community-Supported - This open-source plugin is Couchbase Community supported.
+Note that the Couchbase Lite binary linked to the plugin is Couchbase Lite Enterprise Edition (EE) and its use is guided by the terms and conditions of the https://www.couchbase.com/2018-04-30v3_License_Agreement/[Couchbase Lite Enterprise license agreement].
 
-// == Introduction
+NOTE: _**Couchbase Community Supported**_ is an Open source integration that is developed by Couchbase. Issues related to the integration can be raised in the relevant GitHub repositories or via Couchbase forums or Discord. Support is provided best effort by development team.
```

Comment:
bold AND italic? 😱

Reply:
also did you mean the name of the plugin?
And careful on "Community Supported" vs "Community-Supported" consistency.

Reply:
Removed hyphenation. Keeping the bold AND italic for now as was suggested by PM and is clear/legible in the local docs build.